ev-dev-tools/src/ev_cli/type_parsing.py

```diff
@@ -108,6 +108,10 @@ def generate_tmpl_data_for_type(cls, type_with_namespace, type_def):
 
         type_headers = sorted(helpers.type_headers)
```

Comment:
one could remove this line then and do the list comprehension directly over `sorted(helpers.type_headers)`, right?

Reply:
Yes, if you mean if we can combine line 109 and 113, then yes, but I think its more readable if we don't do 2 things in one line